### DISCUSSION TITLE
Only able to scroll when mouse is hovering.

### DISCUSSION THREAD

**Discussion by OwzyBrand** [LanKuDot/game_modules#49] [Q&A]
Hello! 

I'm using this for a game my development team is making, and I'm trying to make this useable with both controller AND keyboard/mouse. It mostly functions when using the controller but the problem I'm having is this; I can move through the options with the dpad, but it doesn't scroll/ re-arrange the options.
![ezgif-3-26ef349529](https://github.com/user-attachments/assets/a265b504-adb0-472c-a0bd-62b7ecfd7efe)

If there is anyway anyone can think of to make the scrolling list think the mouse is hovering over it while using a controller?

Thanks in advance!

**Comment by LanKuDot** [comment]
Hi, thanks for using the module.
How about storing lists to be controlled and switching between them (such as using RB and LB)?
You may need to highlight the current controlling list. And updating highlighting when using the mouse.

**Reply by OwzyBrand** [reply]
Hey LanKuDot, thanks for the reply!

The way it works now is the player selects a character (the bottom circles from the gif), then it opens the two scroll lists that use this module, and correctly sets the selected object (in the Unity Event System) to the focused box in the list. From there I want it to scroll when I use the dpad on a gamepad, but it currently only does this when I hover my mouse over the area with the circular scroll list.

I'm able to change which list is currently being highlighted/focused but as I said it won't scroll unless the mouse is hovering over it.

Regards,
OwzyBrand

**Reply by LanKuDot** [reply]
You could use `MoveOneUnitUp()` and `MoveOneUnitDown()` to scroll the list.
You need to create a script to get the input of the dpad, and when up arrow, for example, is pressed, invoke the `MoveOneUnitUp()` of the highlighted list.
